src/cell.hpp

```diff
@@ -99,6 +99,11 @@ class cell {
          }
      }
 
+    /// Return the kind of cell, used for grouping into cell_groups
+    cell_kind const get_cell_kind() const  {
```

Comment:
Check, I was thinking this also, but that should be done in a seperate change i guess?